discord/utils.py

```diff
@@ -283,16 +282,15 @@ def decorator(overridden: T) -> T:
     return decorator
 
 
-def warn_deprecated(
+def deprecated_message(
```

Comment:
The git diff makes it look like `warn_deprecated` was removed, but it wasn't see below.